extensions/README.md

```diff
@@ -57,4 +70,4 @@ All extensions used in a tileset or any descendant external tilesets must be lis
 }
 ```
 
-All extensions required to load and render a tileset or any descendant external tilesets must also be listed in the tileset JSON in the top-level `extensionsRequired` array property, such that `extensionsRequired` is a subset of `extensionsUsed`. All values in `extensionsRequired` must also exist in `extensionsUsed`.
\ No newline at end of file
+All extensions required to load and render a tileset or any descendant external tilesets must also be listed in the tileset JSON in the top-level `extensionsRequired` array property, such that `extensionsRequired` is a subset of `extensionsUsed`. All values in `extensionsRequired` must also exist in `extensionsUsed`.
```

Comment:
```suggestion
All extensions required to load and render a tileset or any descendant external tilesets must be listed in the tileset JSON in the top-level `extensionsRequired` array. Extensions in `extensionsRequired` must also be listed in `extensionsUsed`.
```

(Optional) The last sentence of this paragraph seemed redundant, so just rephrasing a bit.